src/app/shell/personal-cabinet/applications/application-card/application-card.component.ts

```diff
@@ -59,7 +59,6 @@ export class ApplicationCardComponent implements OnInit {
       workshopsId: []
```

Comment:
I see you have this **. but I don't see why? Why do you need this value? Remove it if it is not used

Reply:
same for ReasonFormGroup

Reply:
also remove please subrole and its subscription, because it is not used as well